geniza/corpus/management/commands/import_data.py

```diff
@@ -251,14 +257,33 @@ def import_documents(self):
     doctype_lookup = {}
 
     def get_doctype(self, dtype):
-        doctype = self.doctype_lookup.get(dtype, None)
+        # don't create an empty doctype
+        if not dtype.strip():
+            return
+
+        doctype = self.doctype_lookup.get(dtype)
         # if not yet in our local lookup, get from the db
         if not doctype:
             doctype = DocumentType.objects.get_or_create(name=dtype)[0]
             self.doctype_lookup[dtype] = doctype
 
         return doctype
 
+    def get_collection(self, data):
+        lib_code = data.library.strip()
+        # differentiate CUL collections based on shelfmark
+        if lib_code == 'CUL':
+            for cul_collection in ['T-S', 'CUL Or.', 'CUL Add.']:
+                if data.shelfmark.startswith(cul_collection):
+                    lib_code = 'CUL_%s' % cul_collection.replace('CUL ', '')
+                    break
+            # if code is still CUL, there is a problem
+            if lib_code == 'CUL':
+                self.stdout.write(self.style.WARNING(
+                    'CUL collection not determined for %s'
+                    % data.shelfmark))
```

Comment:
idk if we're quite at this point yet, but using the actual `logging` lib has helped me a lot w/ cdhweb exodus, rather than writing to stdout

Reply:
not sure we are either, but thanks for the suggestion, will keep it in mind